datajunction-server/datajunction_server/internal/authentication/github.py

```diff
@@ -58,7 +51,7 @@ def get_github_user(encrypted_access_token: str) -> Optional[User]:  # pragma: n
         pass
     if existing_user:
         _logger.info("OAuth user found")
-        user = existing_user
+        user = existing_user  # type: ignore
```

Comment:
Why do we need `type: ignore` here? Maybe we just need to `cast` `existing_user` to a `User`?

Reply:
That fixed it!